drip/drips.py

```diff
@@ -41,6 +43,32 @@ def message_class_for(name: str):
     return klass
 
 
+class TwilioMessageClient:
```

Comment:
@hadayda Django has a default support for sending emails, so the logic was decoupled implicitly.
I suggest to decouple sending SMS by using Django [Signals](https://docs.djangoproject.com/en/3.1/topics/signals/)